### ISSUE TITLE
another aspect: wishlist

### ISSUE THREAD

**Issue by jancborchardt** [shared-stuff/shared-stuff#7]
In addition to show the stuff you have, it would be good to be able to list the stuff you need. Like @elf-pavlik already did on his website before.

**Comment by shybyte** [comment]
Good idea.

**Comment by shared-stuff** [comment]
This could need a good UI and maybe some rework of existing UI. But for now I will just add it in the simplest possible form. We can add later an awesome UI in a different issue.

**Comment by elf-pavlik** [comment]
great! looking forward :D

**Comment by jancborchardt** [comment]
Yeah, just use the double-radio toggle like on http://twitter.github.com/bootstrap/javascript.html#buttons (give / wish) I’d say.

**Comment by shared-stuff** [comment]
I'm not sure if I really love the bootstrap radio buttons. I will think about it in a future beautification issue, which is needed anyway. For now the simplest possible form are normal radio buttons.

**Comment by jancborchardt** [comment]
The Bootstrap toggle buttons are easier to understand that 2 pure radio buttons next to each other. It’s not about »beautification«, but about the affordance of the interaction element. A toggle button much better visualizes what happens when you click either of the options.

**Comment by shared-stuff** [comment]
At least when I sit outdoor and look at the bootstrap toggle buttons on my dusty laptop screen I have problems to see:
- Are this toggle buttons or action buttons ?
- Can I select one of them or multiple ones? (Radio vs. Checkbox)
- Which one is the selected one ?

And I think: Radio buttons are there for decades and people are just used to it. Even if they are shit, they are well known shit.

Do you have any links to usability research of toggle buttons used as radio buttons? 

I just found this critique http://bugs.jqueryui.com/ticket/7384

**Comment by jancborchardt** [comment]
Yeah, I agree that the blue makes the toggle buttons on the example page pretty shitty. When you remove the »btn-primary« class, they get normal. And in the case of this (give / wish) it’s just two options anyway, so radio vs checkbox does not even come up.

The »well known shit« argument is just plain bad, sorry. Especially if it’s an either-or choice, radio buttons are ridiculous. Fun fact – they are called »radio buttons« because on a radio there are these exclusive buttons which deactivate the other ones. And they look far more like the Bootstrap buttons – and they have been around for way longer than these weird round boxes.

**Comment by shared-stuff** [comment]
The fun fact is great :-) I have to admitted that I missed this fact until now. But one could argue that only very very old people remember the original radio buttons design... (OK, I remember ;-)

Actually I tried to use a double toogle button without btn-primary class. I added an additional active style to differentiate the active button a bit more. Things looked better. But then I found that it looks ugly to have the "give/wish" selection as toogle buttons but the  "Rent/Gift/Use Together" as select boxes. Converting the later one to toggles brought up the problem that it was not obviously anymore that you can select more then one option in the later case. Then I thought: "Fuck it, there are other problems to solve, revolutionizing  the web with remoteStorage might be enough change for today". 

I still wonder if there is scientific research about this theme. All sources I found, recommend to stay safe with standards. 
(http://www.useit.com/alertbox/20040927.html)

**Comment by jancborchardt** [comment]
Ok, so the Rent - Gift - Use together thing is a different problem, I also thought about that.

One thing would be a slider: Gift <- Rent <- Use together
As in – gifting is the most extreme state of sharing. Then you are probably also fine with renting it, or using it together.

But then, I found that I have some stuff which I really want to get rid of, without renting. I guess they should stay checkboxes for now. The metaphor for checkboxes is also just that, a checkbox on paper.

TL;DR: we should use the toggle for Give / Wish, but leave the checkboxes for Gift / Rent / Use together.

**Comment by shared-stuff** [comment]
TLDR
As there is more important stuff to do, I consider this issue as closed. If usability research shows in the future that toggle buttons are superior to radio buttons, we should open another separate issue.
